rts/lra/lra-tck/tck/src/main/java/org/eclipse/microprofile/lra/tck/RunTck.java

```diff
@@ -107,7 +108,7 @@ private void runTck() {
         if (failures.size() != 0) {
             System.out.printf("There were TCK failures:%n");
 
-            failures.forEach(f -> System.out.printf("%s%n", f));
+            failures.forEach(f -> System.out.printf("%s%n", f)); // TODO print out the full test data
```

Comment:
I think we should not add `TODO` into code but rather create an issue on that

Reply:
I think it would be helpful - unless this TODO is already part of JBTM-3089?

Reply:
I already pushed an update to remove this TODO. I will perform a separate analysis to determine exactly which information we want to report.